From 9939271a962e829131a4bebf115ac53b96b69098 Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <akitouni@gnome.org>
Date: Fri, 30 May 2025 15:55:47 +0100
Subject: [PATCH] doc: skip validation of xml

xmllint from the latest release 2.14.3 doesn't considers the includes to be
valid. Older releases (such as 2.12.x used by most distros) are fine. I don't
know if this is a newly implemented check, or if it is buggy behaviour.
---
 doc/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/meson.build b/doc/meson.build
index a03e2ea..9d7a53d 100644
--- a/doc/meson.build
+++ b/doc/meson.build
@@ -14,6 +14,7 @@ xmlto_searchpath = [
 
 xmlto_command = [
   find_program('xmlto'),
+  '--skip-validation',
   '-o', '@OUTPUT@',
   '--searchpath', ':'.join(xmlto_searchpath),
   '-m', files('config.xsl'),
@@ -38,4 +39,4 @@ custom_target(
   depends: xml_dbus_docs,
   install: true,
   install_dir: session_datadir / 'doc' / meson.project_name()
-)
\ No newline at end of file
+)
-- 
2.49.0

